From: SeongJae Park <sj@kernel.org>
Date: Sun, 27 Oct 2024 14:41:50 -0700
Subject: [PATCH] ==== sample DAMON modules ====

mm/damon: add sample modules under samples/damon/

DAMON is a kernel framework that provides API for other subsystems or
modules in the kernel space.  The API functions have their own
kernel-doc comments, and DAMON modules including DAMON_SYSFS,
DAMON_RECLAIM, and DAMON_LRU_SORT can be used as a refernce usages.
However, kernel-doc is not very easy for beginner, and the existing
DAMON modules for real usage are bit complicated.  Add sample DAMON
modules showing basic usage of DAMON kernel API.

Note that some of the code for similar purpose that I used at kernel
summit 2021 DAMON live coding session[1] is reused here.

[1] https://linuxplumbersconf.org/event/11/contributions/984/

Changes from RFC
(https://lore.kernel.org/20241113162624.48973-1-sj@kernel.org)
- Add minimum age to prcl

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/xRddXScV | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/xRddXScV

diff --git a/damon_meta_changes/xRddXScV b/damon_meta_changes/xRddXScV
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

